packages/lesswrong/components/posts/FriendlyPostsItem.tsx

```diff
@@ -0,0 +1,257 @@
+import React, { MouseEvent } from "react";
+import { registerComponent, Components } from "../../lib/vulcan-lib";
+import { AnalyticsContext } from "../../lib/analyticsEvents";
+import { usePostsItem, PostsItemConfig } from "./usePostsItem";
+import { SoftUpArrowIcon } from "../icons/softUpArrowIcon";
+import { HashLink } from "../common/HashLink";
+import { useHistory } from "../../lib/reactRouterWrapper";
+import { SECTION_WIDTH } from "../common/SingleColumnSection";
+import withErrorBoundary from "../common/withErrorBoundary";
+import classNames from "classnames";
+
+export const styles = (theme: ThemeType): JssStyles => ({
+  root: {
+    maxWidth: SECTION_WIDTH,
+    display: "flex",
+    alignItems: "center",
+    background: theme.palette.grey[0],
+    border: `1px solid ${theme.palette.grey[100]}`,
+    borderRadius: 6, // TODO Use theme.borderRadius.default once it's merged
+    padding: `8px 12px 8px 0`,
+    fontFamily: theme.palette.fonts.sansSerifStack,
+    fontWeight: 500,
+    fontSize: 14,
+    color: theme.palette.grey[600],
+    cursor: "pointer",
+    [theme.breakpoints.down("xs")]: {
+      paddingRight: 12,
+    },
+  },
+  karma: {
+    width: 50,
+    minWidth: 50,
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center",
+  },
+  tagRelWrapper: {
+    position: "relative",
+    marginLeft: 30,
+  },
+  voteArrow: {
+    color: theme.palette.grey[400],
+    margin: "-6px 0 2px 0",
+  },
+  details: {
+    flexGrow: 1,
+    minWidth: 0, // flexbox black magic
+  },
+  title: {
+    fontWeight: 600,
+    fontSize: 16,
+    fontFamily: theme.palette.fonts.sansSerifStack,
+    marginBottom: 3,
+  },
+  titleOverflow: {
+    overflow: "hidden",
+    textOverflow: "ellipsis",
+  },
+  meta: {
+    display: "flex",
+    alignItems: "center",
+    "& > :first-child": {
+      flexGrow: 1,
+    },
+  },
+  secondaryContainer: {
+    display: "flex",
+    alignItems: "center",
+  },
+  audio: {
+    "& svg": {
+      width: 15,
+      margin: "3px -8px 0 3px",
+    },
+  },
+  tag: {
+    margin: "0 5px 0 15px",
+  },
+  readTime: {
+    minWidth: 75,
+    textAlign: "right",
+    whiteSpace: "nowrap",
+    paddingRight: 10,
+  },
+  comments: {
+    minWidth: 58,
+    marginLeft: 4,
+    display: "flex",
+    alignItems: "center",
+    "& svg": {
+      height: 18,
+      marginRight: 1,
+    },
+  },
+  bookmark: {
+    minWidth: 20,
+    "&:hover": {
+      opacity: 0.5,
+    },
+  },
+  bookmarkIcon: {
+    fontSize: 18,
+    marginTop: 2,
+    color: theme.palette.grey[600],
+  },
+  hideOnMobile: {
+    [theme.breakpoints.down("xs")]: {
+      display: "none",
+    },
+  },
+  onlyMobile: {
+    [theme.breakpoints.up("sm")]: {
+      display: "none",
+    },
+  },
+});
+
+export type FriendlyPostsListProps = PostsItemConfig & {
+  classes: ClassesType,
+};
+
+const FriendlyPostsItem = ({classes, ...props}: FriendlyPostsListProps) => {
```

Comment:
The tag here is causing the title to be cut quite short, I think we should:
 - Only show a tag chip for core tags
 - Use the `shortName` of the tag

![Screenshot 2023-03-10 at 15 15 48](https://user-images.githubusercontent.com/77623106/224353057-74949d88-1753-4a42-a094-83cd5b8eb3e1.png)

Reply:
think the Shortname version would be great 👍  would be good for "Global health & development" too